test/gtest/common/test_helpers.cc

```diff
@@ -706,4 +708,59 @@ void skip_on_address_sanitizer()
 #endif
 }
 
+bool read_directory(const char *path, bool (cb)(struct dirent *entry, void *ctx),
```

Comment:
can we unite code with ucs_debug_stop_other_threads()?

Reply:
yes, added new calls into ucs/sys